tasks/fix-cat2.yml

```diff
@@ -2376,7 +2466,7 @@
         when:
             - rhel_07_040680_rpm_audit.rc == 0
             - rhel_07_040680_postconf_audit.stdout != 'smtpd_client_restrictions = permit_mynetworks, reject'
-  when: rhel_07_040680
+
```

Comment:
What was the reason for this change? (This is the only one I'd like answered for sure before merging)

Reply:
I didn't intentionally make this one, but I'll put it back in.  Not sure how my copy lost that line.